### DISCUSSION TITLE
iOS Framework documentation?

### DISCUSSION THREAD

**Discussion by Caleb-Bolton** [coqui-ai/STT#2239] [Q&A]
I'm looking for documentation on how to use STT in an iOS app but I can't find any documentation or instructions to get started. 

stt_ios_test in STT/native_client/swift/ does not seem to work out of the box.

Thanks

**Comment by lmille77** [comment]
I also had a difficult time building STT for iOS. The steps I took are:

1. Cloning the project. 
2. Running "git submodule sync tensorflow/" then "git submodule update -init tensorflow/".
3. After that I have been running the "./configure script" and going through the prompt options.
4. Then, I am running the command "bazel build --workspace_status_command="bash native_client/bazel_workspace_status_cmd.sh" -c opt --copt="-D_GLIBCXX_USE_CXX11_ABI=0" //native_client:libstt.so //native_client:generate_scorer_package".
5. Followed by "./tf-setup.sh" and "./ios-build.sh"

Any suggestions on how to get the project to build would be much appreciated.
